extensions/bring/src/index.tsx

```diff
@@ -132,11 +139,12 @@ export default function Command() {
   }, [lists, selectedList, setSelectedList]);
 
   const sections = useMemo(() => {
-    const sections = getSectionsFromData(catalog, listDetail, customItems, translations);
+    if (!listDetail || !("uuid" in listDetail) || !Array.isArray(customItems)) return [];
+    const sections = getSectionsFromData(catalog, listDetail as BringList, customItems, translations);
     return addNewItemToSectionBasedOnSearch(sections, search, translations);
   }, [catalog, listDetail, customItems, translations, search]);
 
-  if (!selectedList) {
+  if (!selectedList || purchaseStyle === null) {
```

Comment:
This change breaks the previous caching behaviour. You notice it when you run the command. Before it would immediately display the list based on the cached state from the previous run and then update it as soon as we get a response from the API. Now it will just stay empty during that period.